database/mysql.go

```diff
@@ -168,6 +173,8 @@ func mysqlConnection(logger log.Logger, mysqlConfig *MySQLConfig, databaseName s
 						return nil
 					}
 				}
+
+				tlsConfig.Certificates = []tls.Certificate{&clientCert}
```

Comment:
What if we add that cert when it's not nil?
```suggestion
				if clientCert != nil {
					tlsConfig.Certificates = append(tlsConfig.Certificates, *clientCert(
				}
```

Reply:
I changed the `LoadTLSClientCertFromEnv` function to instead return a non-pointer Certificate.  I figure since I'm checking for the error, it should handle the error case.  But let me know if you think it's better to use a pointer and check for nil as you suggested.